### PR TITLE
Fixing of stats tests

### PR DESCRIPTION
**Pull request by cemsbr**
- Split of_stats/main.py in several files
- Make it possible to import napps from working tree, so linter works properly

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,18 +23,18 @@
     BASE_ENV = '/'
 
 NAPPS_PATHS = {
-    'main': os.path.join(BASE_ENV, 'var/lib/kytos/napps')
+    'main': join(BASE_ENV, 'var/lib/kytos/napps')
 }
-NAPPS_PATHS['installed'] = os.path.join(NAPPS_PATHS.get('main'), '.installed')
-NAPPS_PATHS['enabled'] = os.path.join(NAPPS_PATHS.get('main'), 'kytos')
+NAPPS_PATHS['installed'] = join(NAPPS_PATHS.get('main'), '.installed')
+NAPPS_PATHS['enabled'] = join(NAPPS_PATHS.get('main'), 'kytos')
```
Comment (beraldoleal) [inline setup.py]: Why did you changed this ? It was some linter error ?

Comment (cemsbr) [inline setup.py]: - Linter errors: import order, no space after comma;
- Consider only folders in `retrieve_apps` (I had to add the file `__init__.py`);
- Removed `os.path` in the code to make it more readable.

Comment (beraldoleal) [inline setup.py]: But the os.path is not a linter error, right ? So join is to much generic.. os.path.join makes more sense than only join.

Comment (cemsbr) [inline setup.py]: No, it is not a linter error. `join` is not a built-in function and I see no ambiguity in its meaning. Anyway, I changed to `os.path.join`.

**Pull request merged**